BridgeSDK/Networking/SBBNetworkManager.m

```diff
@@ -218,6 +218,11 @@ - (instancetype) initWithBaseURL: (NSString*) baseURL bridgeStudy: (NSString*)br
         self.environment = SBBEnvironmentCustom;
```

Comment:
Is this right? Does it get reset somewhere else? I realize its not part of the PR changes but something I noticed